mapdata/WarcraftLegacies/AbilityData/Skin/843460673.json

```diff
@@ -1,40 +1,45 @@
 {
   "OldId": 1986085697,
-  "NewId": 1194537281,
+  "NewId": 843460673,
   "Unk": [
     0
   ],
   "Modifications": [
     {
-      "Id": 1718840929,
+      "Id": 1953653089,
       "Type": 3,
-      "Value": "(Sunfury - Void Reaver)"
+      "Value": "ReplaceableTextures\\CommandButtons\\PASUnholy.blp"
     },
     {
-      "Level": 1,
-      "Id": 828536161,
+      "Id": 2020631137,
+      "Value": 1
+    },
+    {
+      "Id": 1718840929,
       "Type": 3,
-      "Value": "The massive Void Reaver demoralize the enemy units reducing their armor by 20%"
+      "Value": ""
     },
     {
       "Id": 1835101793,
       "Type": 3,
-      "Value": "Aura of the Reaver"
+      "Value": "Vulnerability Aura(Sunfury- Void Reaver)"
     },
     {
-      "Id": 1953653089,
+      "Id": 1952543841,
       "Type": 3,
-      "Value": "ReplaceableTextures\\CommandButtons\\BTNPossession.blp"
+      "Value": ""
     },
     {
-      "Id": 2020631137,
-      "Value": 1
+      "Level": 1,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Vulnerability Aura"
     },
     {
       "Level": 1,
-      "Id": 829453409,
+      "Id": 828536161,
       "Type": 3,
-      "Value": "Aura of the Reaver"
+      "Value": "Reduces the Armor of nearby hostile units by -3."
```

Comment:
Armor shouldn't be capitalized; not a proper noun. A reduction of -3 would be an increase of 3. So this should be "Reduces the armor of nearby hostile units by 3.".